scripts/plot_joined_scalars.py

```diff
@@ -31,7 +31,7 @@
 def prepare_scalar_data(df, colors_odict, labels_dict, conv_number):
     # pivot
     df_pivot = pd.pivot_table(
-        df, index=["scenario", "region"], columns="name", values="var_value"
+        df, index=["scenario_key", "region"], columns="name", values="var_value"
```

Comment:
The postprocessed data are labeled by scenario, not scenario_key.

Reply:
Done in [0da658b](https://github.com/rl-institut/oemof-B3/pull/147/commits/0da658bd4567e29e1929d6f1cb95a9459975a051)